content/en/austin/get_involved/current_issues/i35_expansion.md

```diff
@@ -0,0 +1,47 @@
+
+TxDOT has yet to listen to the public, to elected officials, and to common sense that shows [highways have been a fatal and failed experiment](/austin/transportation/highways) in moving people safely and efficiently around the US.
+
+## Rethink35 Lawsuit
+
+- Rethink35 and 16 other plaintiffs have sued TxDOT. The lawsuit alleges:
+  - TxDOT didn't properly study other options such as:
+    - [The Rethink 35 proposal](https://rethink35.org/proposal) which involved rerouting the highway and replacing it with a boulevard.
+    - [The Reconnect Austin vision](https://reconnectaustin.com/#home-vis-sec) which called for the highway being placed underground with a boulevard on top.
+  - TxDOT failed to study several air pollutants and their impacts on the people who live near I-35. This includes particulate matter 2.5 (PM2.5) which puts people at increased risk for asthma and other health conditions. Based on new EPA standards, Austin is already above the safe limit.
+  - There was a significant change to the water outfall that was made after the Draft Environmental Impact Statement was published and so there was no opportunity for public comment. Polluted runoff from the highway will exit into a wildlife preserve in East Austin; around US-183 and the Colorado River.
+
+{{% alert title="Highways as segregation tools" color="primary" %}}
+Highway construction and expansion is one of many tools used to systematically segregate our cities. The [history of housing segregation](/austin/housing/housing_segregation) is as brutal as it is banal. These injustices persist to the present day.
+{{% /alert %}}
+
+## Civil Rights (Title VI) Complaint
+
+Rethink35 has also [filed a Title VI civil rights complaint](https://www.documentcloud.org/documents/24390031-rethink35-title-vi-complaint-filed). The complaint alleges:
+
+- I-35 was built to be intentionally discriminatory by replacing East Ave and solidifying a line of segregation between East and West Austin.
+- Under Title VI, agencies with a history of prior discrimination have a responsibility to take affirmative action to redress previous injustices.
+- Instead, this project exacerbates these very same harmful practices by:
+  - Displacing people from homes and businesses. 86% of the displaced are in Environmental Justice communities (ie. low income, and people of color).
+  - More than doubling the number of lane miles of I-35. Thus widening the barrier between East and West Austin.
+  - Dumping polluted storm water into a Wildlife Sanctuary in East Austin, continuing a long-standing pattern of moving environmental waste from downtown to East Austin.
+  - Worsening air quality, climate change, and increasing summer temperatures, all of which disproportionately burden communities of color.
```

Comment:
@kelseyhuse30 Source on these things disproportionately impacting communities of color? I don't doubt it but I like having sources for people to learn more :slightly_smiling_face:

Reply:
https://docs.google.com/document/d/1Kr_qSeNtVjSU1Fe69Kog6HDyk9P7rz0l0WVXdhcsRYU/edit

I think it's somewhere in here